spec/acceptance/pkcs12_spec.rb

```diff
@@ -1,16 +1,19 @@
 require 'spec_helper_acceptance'
 
+# rubocop:disable RSpec/InstanceVariable : Instance variables are inherited and thus cannot be contained within lets
+
 # SLES by default does not support this form of encyrption.
-describe 'managing java pkcs12', unless: (UNSUPPORTED_PLATFORMS.include?(host_inventory['facter']['os']['name']) || host_inventory['facter']['os']['name'] == 'SLES') do
-  # rubocop:disable RSpec/InstanceVariable : Instance variables are inherited and thus cannot be contained within lets
+describe 'managing java pkcs12', unless: (UNSUPPORTED_PLATFORMS.include?(os[:family]) || os[:family] == 'SLES') do
+  def keystore_command(target)
```

Comment:
potential ?
`def keystore_command
   command = "\"#{@keytool_path}keytool\" 
   command.prepend('& ') if os[:family] == 'windows'
   command
`

Reply:
I'v refactored this function out of the spec files and into the spec_helper_acceptance_local. See what you think.